tests/scripts/github-action-helper.sh

```diff
@@ -133,6 +133,7 @@ function validate_yaml() {
   cd cluster/examples/kubernetes/ceph
   kubectl create -f crds.yaml -f common.yaml
   # skipping folders and some yamls that are only for openshift.
+  # shellcheck disable=SC2010,SC2046 # Safe to grep with ls
```

Comment:
It's fine to use `ls` if it's the only option, but why quietly ignore SC2046 with no comment?

`find` is a better tool for this anyway. And splitting into multiple lines makes it easier to read.

- `ls -p | grep -v /` is a hacky way to avoid directories when `find -type f` is straightforward.
- `-maxdepth 1` makes it clear that there is no recursion happening, which is implicit with `ls`.
- `-name *.yaml` makes it clear we are only choosing .yaml files, and there is no need to ignore .sh or .py files, or any other type of file that makes it into that dir in the future.
- `-and -not -name '*openshift*'` is a nice human readable way to show we are ignoring openshift-related manifests.

```bash
manifests="$(find . -maxdepth 1 -type f -name '*.yaml' -and -not -name '*openshift*')"
with_f_arg="$(echo "$manifests" | awk '{printf " -f %s",$1}')" # don't add newline
# shellcheck disable=SC2046 # '-f manifest1.yaml -f manifest2.yaml etc.' should not be quoted
kubectl create ${with_f_arg} --dry-run
```